pom.xml

```diff
@@ -25,8 +25,8 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling</artifactId>
-        <version>31</version>
-        <relativePath/>
+        <version>34</version>
+        <relativePath>../../../parent/pom.xml</relativePath>
```

Comment:
Why did you add this change?

Reply:
I thought it is good practice to always go with the latest parent - if You're refering to the relative path this is a merge artifact and regression as my initial patch was against an older version so could be dropped.

Reply:
Yes, I mean the relativePath. Thanks for dropping.